src/rotation/log_new.zig

```diff
@@ -0,0 +1,340 @@
+const std = @import("std");
```

Comment:
Before reviewing this, I would like to know what the behavioral differences to the branch rotation are?
Could we just use a branch rotation with diameter 1 instead of adding a new rotation mode?

Reply:
- it shows a secondary texture when broken,
- it preserves the shape after breaking neighbor blocks,
- diameter of 1 may screw with how extending works in branches